src/Games/brain-even.js

```diff
@@ -0,0 +1,18 @@
+import getRandomNumber from '../randomCount.js';
+import theCoreOfGames from '../index.js';
+
+const brainRulls = () => {
+  const descriptionOfGame = 'Answer "yes" if the number is even, otherwise answer "no".';
+  const randomNumber = getRandomNumber(1, 100);
+  const correctAnswer = randomNumber % 2 === 0 ? 'yes' : 'no';
+  return [descriptionOfGame, randomNumber, correctAnswer];
+};
+
+const preparedGame = () => {
```

Comment:
brainRulls

Reply:
dont understand the question...

Reply:
I meant to rename the function to `even`

Reply:
ok, fixed